pvm.py

```diff
@@ -30,12 +38,7 @@ def _init_logger():
 _logger = logging.getLogger("PVM")
 _logger.info("Logging system initiated in %s", LOG_PATH)
 
-# Place your videos in this folder for autostart
-LOG_PATH = "/home/pi/PVM/log/"
-PEFIX_PATH = "/home/pi/Videos/"
-VIDEO_PATH = "jellyfish720p.mp4"
 media = ""
```

Comment:
going to leave `media = ""` all alone here?
😄

Reply:
Yeah, good question. My rationale is that LOG_PATH, PEFIX_PATH, and VIDEO_PATH are global constants that will never change, but `media` is a global variable that will have different values. Since they are different types of variables I think it'd be better to separate them, and also constants usually come first in a file rather than in the middle (according to my experience).

Reply:
> My rationale is that LOG_PATH, PEFIX_PATH, and VIDEO_PATH are global constants that will never change,

Correction, `VIDEO_PATH` is constantly changing...

> also constants usually come first in a file rather than in the middle (according to my experience).

exactly. I like that you moved things up!